src/mon_apl.js

```diff
@@ -723,21 +690,21 @@
               insertTextRules,
               documentation: 'Property declaration',
             },
-            {
-              label: 'Section',
-              kind: kind.Snippet,
-              insertText: [
-                'Section ${1:name}',
-                '\t$0',
-                ':EndSection',
-              ].join('\n'),
-              insertTextRules,
-              documentation: 'Section block',
-            },
           );
         }
         // if (!t || t === '∇') {
         suggestions.push(
+          {
+            label: 'Class',
+            kind: kind.Snippet,
+            insertText: [
+              'Class ${1:name}',
+              '\t$0',
+              ':EndClass',
+            ].join('\n'),
+            insertTextRules,
+            documentation: 'Class script',
+          },
```

Comment:
I just added it as a default suggestion, however, that means that if you're inside a class, then `:c` would still suggest `:Class`, which may not be desired. We can hide it behind an `if (t !== 'class')` to not allow nesting. Again though, I'm not well-versed in APL to know if this is allowed or not allowed. 

Same goes for the other snippets.